README_VERITABLE.md

```diff
@@ -276,7 +276,7 @@ docker run -it \
   -e ALICE_BASE_URL=http://alice:3000 \
   -e BOB_BASE_URL=http://bob:3000 \
   -e CHARLIE_BASE_URL=http://charlie:3000 \
-  afj-rest-integration-tests
+  afj-rest-integration-tests npm run test:integration
```

Comment:
so if `npm run test:integration` was missing up until now how was it running?

Reply:
It was running unit tests as the default